.scala-steward.conf

```diff
@@ -2,8 +2,8 @@ updates.pin = [
     { groupId = "org.scala-lang", artifactId = "scala-library", version = "2.12." }
 ]
 
-# updates.ignore = [
-#     { groupId = "org.apache.spark", artifactId = "spark-sql", version = "3." }
-# ]
+updates.ignore = [
+    { groupId = "org.scoverage", artifactId = "sbt-scoverage" }
```

Comment:
```suggestion
# Ignored because of spark 2.4 as there is no version for scala 2.11 on further versions
    { groupId = "org.scoverage", artifactId = "sbt-scoverage" }
```